src/logger.ts

```diff
@@ -15,4 +18,5 @@ export function log(message: string, event?: string){
     logMessage += `${message}`;
 
     console.log(logMessage);
+    extensionVariables.outputChannel.appendLine(logMessage);
```

Comment:
what is this for?

Reply:
This is to output message to Azure-Pipelines channel in the output section of vs code

Reply:
Dis this as console.log, only works when we are running extension in debug mode. In this case, console.log prints the message to output section of the host vscode window where F5 is pressed

Reply:
Then do we need console.log()?

Reply:
Console.log prints output on the host windows output section. 
It can be removed as well now that we are printing to output channel: Azure-Pipelines.

However, console.log will still be helpful when extension loading starts failing, then we would not have output channel. In the case we can get debug logs from console.log.

@bishal-pdMSFT what do you think, should we remove ?

Reply:
You mean activate failures? We will get the telemetry for such failure through `callWithTelemetryAndErrorHandling`. No?

Reply:
Yeah that's true. console.log is majorly for debugging purpose while in development mode. 
But i see the use case is very small hence removing it now.